docs/PROPERTIES.md

```diff
@@ -250,3 +253,107 @@ Example of the `slides` control type:
         ]
     }
 ```
+
+## Conditional child properties
+
+Conditional child properties provides to the ability to present to the user a set of properties (child properties) depending on the value of another property (parent property)
+
+The `childProperties` property is a list of combinations of a condition and a list of properties. If the given condition matches the value of the parent property, the given list of properties are displayed as child properties.
```

Comment:
`is a list of combinations of a condition and a list of properties` -> `is a list of combinations consisting of a condition and another list of properties` maybe? Otherwise it seems the last `and a list of properties` might be separate?

Reply:
Updated, please check